src/openvpn.mli

```diff
@@ -246,6 +246,22 @@ module Config : sig
   (** Parses a configuration string, looking up references to external files
       as needed. Validates the client configuration. Default client options are
       applied. *)
+
+  val parse_server : string_of_file:(string -> (string, R.msg) result) ->
+    string -> (t, [> R.msg]) result
+  (** Parses a configuration string, looking up references to external files
+      as needed. Validates the server configuration. Default server options are
+      applied. *)
+
+  val a_ca_payload : string -> (b, string) result
+  (**  for test *)
+  
+  val a_cert_payload : string -> (b, string) result
+  (**  for test *)
+
+  val a_key_payload : string -> (b, string) result
+  (**  for test *)
```

Comment:
I prefer to not export these bindings. I know this means that we can "only" test a "full" configuration, which is fine with me. The amount of stuff in `Openvpn.mli` should be minimal.

Reply:
I meant any of the above, I'm more in favour to potentially replicate some code in the test library rather than have these bindings here. to me, it looks like X509.Certificate.decode_pem etc. already do what needs to be done here (apart from the static key/tls-auth hex-decoding and split into four sections).

Reply:
Ok, so i copied those functions into the test

Reply:
thanks, that looks much cleaner to me.